Command/GenerateCommand.php

```diff
@@ -29,7 +29,7 @@ class GenerateCommand extends ContainerAwareCommand
     /**
      * {@inheritdoc}
      */
-    protected function configure()
+    protected function configure(): void
```

Comment:
Can we do this on implementations of symfony interfaces?

Reply:
Why not?

Reply:
Thought this wasn't possible, but it is oO
https://3v4l.org/pX4G9

... unless symfony won't add return types
https://3v4l.org/SnXSO

Reply:
Yes indeed, but that's very unlikely, isn't it?

Reply:
They couldn't do this in a minor release, so okay for now